go/tao/rollback.go

```
@@ -79,11 +72,187 @@ func Unprotect(keys []byte, in []byte) ([]byte, error) {
 	hm.Write(in[32:])
 	calculatedHmac := hm.Sum(nil)
 	if bytes.Compare(calculatedHmac, in[0:32]) != 0 {
-		return nil, errors.New("Unprotect: Bad mac")
+		return nil, errors.New("unprotect: Bad mac")
 	}
 	return out, nil
 }
 
+// Initialize rollback data.
+func InitRollbackState(tableName string, sealsBeforeSave int) {
+	counterTableInitialized = false
+	numSealsBeforeSave = sealsBeforeSave
+	numSeals = 0
+	hostRollbackTableFileName = &tableName
+}
+
+// Read the counter table.
+func ReadRollbackTable(fileName string, tableKey []byte) *RollbackCounterTable {
+	blob, err := ioutil.ReadFile(fileName)
+	if blob == nil || err != nil {
+		return new(RollbackCounterTable)
```

Any reason we want to swallow the error here and not return nil?